config/packages/cache.yaml

```diff
@@ -16,3 +16,20 @@ framework:
                 adapter: cache.adapter.redis
                 # Default expire set to 5 minutes
                 default_lifetime: 300
+
+            # Creates a "auth-screen.cache" service
+            authscreen.cache:
+                adapter: cache.adapter.redis
+                # Default expire set to 1 hour
+                default_lifetime: 3600
+
+when@test:
```

Comment:
We should decide when/if to use this as opposed to having independent yaml files i.e `/config/packages/test/cache.yaml`